spec/support/service_spec_helper.rb

```diff
@@ -21,11 +21,22 @@ def default_headers
       'x-rh-insights-request-id' => 'gobbledygook' }
   end
 
+  def modified_headers(user_hash, request_id=nil)
```

Comment:
@syncrou Do we need this. Our default user hash has things we need

Reply:
We don't have a nice way to create an org_admin from the `default_user_hash`, this could probably be moved to the common gem, but I needed an `org_admin` passed in so created these helpers to accomplish that.

Reply:
@syncrou With the way its implemented won't be allowing non admin users to do the seeding. We only want the org_admin to do the seeding all others should fail. If that's the case then we shouldn't have to redo any of the hash logic.

Reply:
The api though will allow none `org_admin` users to try though. I do need more tests using our `default_headers` which do not include an `org_admin`.